nodepool/scripts/prepare_node.sh

```diff
@@ -23,6 +23,11 @@ THIN=${THIN:-true}
 ALL_MYSQL_PRIVS=${ALL_MYSQL_PRIVS:-false}
 GIT_BASE=${GIT_BASE:-git://git.openstack.org}
 
+# delete stack user if exists, in case the base XVA has this user already.
+if id stack ; then
```

Comment:
yes. If the stack exists, it will fail the stack.sh which will try to create this user. If it failed creating this user, it will be topped in the middle.